torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -414,6 +414,13 @@ static const std::vector<OperatorGeneratorArgs> opGenArgs{
         TORCH_SELECTIVE_SCHEMA("aten::sym_size(Tensor self) -> SymInt[]"),
         sym_size,
         aliasAnalysisFromSchema()),
+    OperatorGeneratorArgs(
+        TORCH_SELECTIVE_SCHEMA("aten::stride(Tensor self) -> int[]"),
+        [](Stack& stack) {
+          at::Tensor arg = pop(stack).toTensor();
+          push(stack, arg.strides());
```

Comment:
Does this need to be `arg.strides.vec()`? It seems like this should be fine... but all of the other methods for e.g. `size` and `sym_size` (defined in `promoted_prim_ops.cpp`) end up `.vec()`'ing the `IntArrayRef` before putting it into the stack.

Reply:
Stack is owning so it indeed should be Ok, though it might be better to follow the prevailing convention